.editorconfig

```diff
@@ -0,0 +1,2 @@
+[*.hcl]
+indent_size = 2
```

Comment:
Shouldn’t this be enforced by terraform fmt?

Reply:
This file is to inform IDEs and text editors about our preferred format. Additionally, `terraform fmt` only formats `.tf` files, not `.hcl`, and this file is effectively just a template file as far as terraform is concerned.

Reply:
True! Did not realize `.hcl` is for Nomad job specs.